tests/django/test_dataloaders.py

```diff
@@ -10,15 +10,15 @@
 try:
     import django
 
-    DJANGO_VERSION: tuple[int, int, int] = django.VERSION
+    DJANGO_VERSION: str = django.get_version()
 except ImportError:
-    DJANGO_VERSION = (0, 0, 0)
+    DJANGO_VERSION = "0.0.0"
 
 
 pytestmark = [
     pytest.mark.asyncio,
     pytest.mark.skipif(
-        DJANGO_VERSION < (3, 1),
```

Comment:
I really prefer tuples comparison, as strings would fail for `"3.10" > "3.2" for example.

Any reason for this change

Reply:
You're right. I reverted it